apps/cli/app.ts

```diff
@@ -360,12 +385,11 @@ class EdgeApp {
       
       // Start OAuth server immediately for easy access
       const oauthPort = parseInt(process.env.CYRUS_OAUTH_CALLBACK_PORT || '3457', 10)
-      const oauthCallbackBaseUrl = process.env.CYRUS_OAUTH_CALLBACK_BASE_URL || `http://localhost:${oauthPort}`
       if (!this.oauthServer) {
         this.startOAuthServer(oauthPort)
         console.log(`\n🔐 OAuth server running on port ${oauthPort}`)
```

Comment:
@Connoropolous is this log correct still if you pass in a different callback url and no port